apps/api-example/src/app/index.ts

```diff
@@ -74,32 +75,35 @@ export async function runServer(app?: Application | undefined) {
   if (
     process.env.MAILGUN_API_KEY &&
     process.env.MAILGUN_BASE_DOMAIN &&
-    process.env.MAILGUN_MAIL_DOMAIN &&
-    process.env.MAILGUN_WEBHOOK_SECRET
+    process.env.MAILGUN_MAIL_DOMAIN
   ) {
     mailProvider = new MailgunMailProvider({
       id: 'mailgun',
       name: 'Mailgun',
-      fromAddress: 'dev@wepublish.ch',
+      fromAddress:
+        process.env.MAILGUN_FROM_ADDRESS || process.env.DEFAULT_FROM_ADDRESS || 'dev@wepublish.ch',
       webhookEndpointSecret: process.env.MAILGUN_WEBHOOK_SECRET,
       baseDomain: process.env.MAILGUN_BASE_DOMAIN,
       mailDomain: process.env.MAILGUN_MAIL_DOMAIN,
       apiKey: process.env.MAILGUN_API_KEY,
       incomingRequestHandler: bodyParser.json()
     })
   }
-  // left here intentionally for testing
-  /* if (process.env.MAILCHIMP_API_KEY && process.env.MAILCHIMP_WEBHOOK_SECRET) {
+
+  if (process.env.MAILCHIMP_API_KEY) {
```

Comment:
webhook isn't really needed, only adds it to the maillog when a mail is sent through Mailchimp directly

Maillog is not used anywhere